libs/core/linearSolverCG.cpp

```diff
@@ -115,7 +117,9 @@ int cg::Solve(solver_t& solver,
     //  x <= x + alpha*p
     //  r <= r - alpha*A*p
     //  dot(r,r)
+    if (MAXIT <= 100) roctxRangePush("cg::UpdateCG");
```

Comment:
It's a little surprising that you're annotating the MPI comms, and the CG update, but nothing else? Any reason the CG update is so special?